tests_integration/__tests__/patterns.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const runPrettier = require("../runPrettier");
+
+expect.addSnapshotSerializer(require("../path-serializer"));
+
+// TODO: move `multiple-patterns` tests into this one
```

Comment:
Can you explain why we would want to do this in the future?

I would only favor combining tests if they have the exact same output and can help reduce the number of snapshots we have to maintain.

Assuming there would be a good reason to do this, I would really favor adding this comment to `tests_integration/__tests__/multiple-patterns.js` as well to avoid losing track of this TODO item.

Reply:
not all tests are `multiple-patterns` , name to `patterns` should be more reasonable